Package.swift

```diff
@@ -90,6 +90,10 @@ let package = Package(
       ],
       linkerSettings: swiftSyntaxParserLinkerSettings
     ),
+    .target(
+      name: "SwiftSyntaxSupport",
```

Comment:
Shouldn’t this be named `SwiftSyntaxTestSupport` if it imports `XCTest`?

Reply:
Yeah, probably should 👍